.circleci/config.yml

```diff
@@ -321,41 +337,6 @@ commands:
             done;
           #
 
-  conditionally-launch-build-and-store:
-    description: Launch build and save job if build missing
-    parameters:
-      study_key:
-        type: string
-    steps:
-      - setup-shared-env:
-          study_key: << parameters.study_key >>
-      - run:
-          name: Initiate build for <<parameters.study_key>> and git SHA << pipeline.git.revision >> if existing one not found
-          command: |
-            set -u
-            set +e
-            readvault.sh secret/pepper/prod/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
-            TAR_FILE_URL_PATTERN="gs://${BUILDS_BUCKET}/${ANGULAR_PROJECT_NAME}/${ANGULAR_PROJECT_NAME}_*_${SHORT_GIT_SHA}.tar.gz"
-            echo "Checking for ${TAR_FILE_URL_PATTERN}"
-            # For some reason following line generating exit code 1 every time in CircleCI. Use set +e so script can continue
-            TAR_FILE_URL=`gsutil ls $TAR_FILE_URL_PATTERN  | head -1`
-
-            # if not found, let's kick off a build
-            if [ -z $TAR_FILE_URL ]
-              then
-                echo "No build found for <<parameters.study_key>> with git sha ${SHORT_GIT_SHA}. Initiating a build for it."
-                curl -u "${CIRCLE_API_TOKEN}:" -X POST --header "Content-Type: application/json" -d '{
-                  "branch": "<< pipeline.git.branch >>",
-                  "parameters": {
-                      "study_key": "<<parameters.study_key>>",
-                      "do-builds": true
-                  }
-                }' https://circleci.com/api/v2/project/gh/broadinstitute/ddp-angular/pipeline
```

Comment:
So this launches a separate CircleCI workflow to do the build for a study. That workflow is `build-app-workflow`. Because it's a separate workflow, it's not easy to find out status of new workflow: when it has started, passed or failed. Only way to find out workflow status is by calling other CircleCI api which would make config much more complicated.